src/ParallelDataCollector.cpp

```diff
@@ -956,6 +970,45 @@ namespace splash
         dataset.close();
     }
 
+    CollectionType* ParallelDataCollector::readDataSetMeta(H5Handle h5File,
+            int32_t id,
+            const char* name,
+            const Dimensions dstBuffer,
+            const Dimensions dstOffset,
+            const Dimensions srcOffset,
+            Dimensions &sizeRead,
+            uint32_t& srcDims)
+    throw (DCException)
+    {
+        log_msg(2, "readDataSetMeta");
+
```

Comment:
probably it would be easier to read in our venerable age if we would add more ascii art to the blocks. kids these day...!